README.md

```diff
@@ -1,3 +1,42 @@
-# resin.io docs
+Resin.io Documentation
+======================
 
-This repo contains our public-facing documentation.
+[![dependencies](https://david-dm.org/resin-io/docs.png)](https://david-dm.org/resin-io/docs.png)
+[![Build Status](https://travis-ci.org/resin-io/docs.svg?branch=master)](https://travis-ci.org/resin-io/docs)
```

Comment:
These two "build status" badges should be removed, given that this repository doesn't run any tests in the CI servers.